iothub/device/src/Twin/PropertyCollection.cs

```diff
@@ -100,7 +101,7 @@ public bool TryGetValue<T>(string propertyKey, out T propertyValue)
         /// </summary>
         public string GetSerializedString()
         {
-            return PayloadConvention.PayloadSerializer.SerializeToString(_properties);
+            return JsonConvert.SerializeObject(_properties);
```

Comment:
can we do this by referencing the default convention?

Reply:
We lost the half step of going from an object to a string, and not straight to bytes.

I could expose that on the default one, though.